docs/source/conf.py

```diff
@@ -14,10 +14,10 @@
 # -- General configuration ---------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
 
-extensions = []
+extensions = []  # type: list[str]
 
 templates_path = ["_templates"]
-exclude_patterns = []
+exclude_patterns = []  # type: list[str]
```

Comment:
mypy not supposed to run here, thanks to exclusion pattern in pre-commit config
```suggestion
exclude_patterns = []
```